src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -38,11 +38,23 @@ trait GoogleServicesDAO {
 
   def isUserInProxyGroup(user: RawlsUser): Future[Boolean]
 
+  def createGoogleGroup(groupName: String): Future[Unit]
+
+  def addMemberToGoogleGroup(groupName: String, memberEmail: String): Future[Unit]
+
+  def removeMemberFromGoogleGroup(groupName: String, memberEmail: String): Future[Unit]
+
```

Comment:
as noted in other comments, add and remove are higher order functions and should be at the service level.